### ISSUE TITLE
Comments: untitled posts create display issues

### ISSUE THREAD

**Issue by drw158** [Automattic/wp-calypso#19486]
![screen shot 2017-11-03 at 2 20 31 pm](https://user-images.githubusercontent.com/618551/32392239-cdbf981a-c0a2-11e7-8aaa-4de64b256194.png)

Notice how some of these comments are on untitled posts (photo posts). For some reason, some display `Untitled` some are just blank? I don't know why it's blank or Untitled in some places.

Notice the title in the single post view:

![screen shot 2017-11-03 at 2 23 10 pm](https://user-images.githubusercontent.com/618551/32392453-cd099d16-c0a3-11e7-9247-8aea9511c0fd.png)

And just blank while expanded:

![screen shot 2017-11-03 at 2 24 38 pm](https://user-images.githubusercontent.com/618551/32392473-e1d8ac78-c0a3-11e7-800f-647b31fc9d1d.png)

**Comment by Copons** [comment]
This might be related to https://github.com/Automattic/wp-calypso/issues/18993

Are those your own posts @drw158, aren't them? Could you check if by any chance the titles are actually invisible characters somehow?

**Comment by drw158** [comment]
I played around with it, and I was able to "fix" it by going the the editor, switching to the html tab, then saving. Without actually changing anything. From what I could tell, those posts became blank because:

- both posts had no title (obviously)
- one post had a blank line above the image in the visual editor. html editor just had an <a> tag wrapping an image.
- one post just had an <img> tag in the html editor with no <a> tag.

Switching the editor tabs around and resaving solved the issue.

Also, weirdly, I was able to click on the blank space title in Comments probably because it was a blank line. On another post, the link to the post was completely not there (not able to click it).

I can try to reproduce if you want!
